copy_assets.sh

```
@@ -0,0 +1,43 @@
+#!/bin/bash
+
```

`set -eo pipefail` to fail fast